cinema/migrations/0001_initial.py

```diff
@@ -73,6 +76,9 @@
                 ),
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
+                ("duration", models.IntegerField()),
+                ("actors", models.ManyToManyField(to="cinema.actor")),
```

Comment:
The `actors` field in the Movie model uses a ManyToManyField to the Actor model, allowing a movie to have multiple actors. This is correctly implemented.